simulator-ui/src/main/resources/static/src/app/components/scenarios/launch/scenario-launch.ts

```diff
@@ -30,18 +30,18 @@ export class ScenarioLaunchComponent implements OnInit {
 
     getScenario(name: string) {
         this.scenarioService.getScenario(name)
-            .subscribe(
-                scenario => this.scenario = scenario[0],
-                error => this.errorMessage = <any>error
-            );
+            .subscribe({
+                next:scenario => this.scenario = scenario[0],
+                error:error => this.errorMessage = <any>error
+            });
```

Comment:
actually, what was the problem with the previous version? I thought this would still work as expected..?